pkg/task/identity_test.go

```diff
@@ -23,26 +23,354 @@ import (
 // TODO
 func TestNewTaskIdentifier(t *testing.T) {}
 
-// TODO
-func TestIsPod(t *testing.T) {}
+func TestIsCommand(t *testing.T) {
+	tests := map[string]struct {
+		taskIdentifier taskIdentifier
+		isCommand      bool
+	}{
+		"kindCommand#1": {
+			taskIdentifier: taskIdentifier{
+				identity: MetaTaskIdentity{
+					Kind: "Command",
```

Comment:
What about adding a test case e.g. `Commands` ( to check for plural kind: whether `Commands` is a valid command type or not?)
Similarly for all other test functions below.